### PR TITLE
Reut/feat/logger

### PR DESCRIPTION
**Pull request by ReaGol**

### CODE REVIEW

File: services/logger/prodLogger.js [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+function buildProdLogger() {
+	return createLogger({
+		level: 'debug',
+		format: combine(timestamp(), errors({ stack: true }), json(), colorize()),
+		levels,
+		defaultMeta: { service: 'user-service' },
+		transports: [
+			new transports.Console(),
+			new transports.File({ filename: 'logs.js', level: 'error' }),
```
Comment (pini85) [inline services/logger/prodLogger.js:20]: why is this logs.js if the file output should be logs.json?

File: services/logger/prodLogger.js [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+const levels = {
+	error: 0,
+	warn: 1,
+	info: 2,
+	http: 3,
+	debug: 4,
+};
+
+function buildProdLogger() {
```
Comment (pini85) [inline services/logger/prodLogger.js:12]: better to create a variable called Logger
want invoke it already here and return it
like const Logger = winston.createLogger(..settings)
export default Logger

File: services/logger/prodLogger.js [hunk 3]
```diff
@@ -0,0 +1,25 @@
+import { format, createLogger, transports } from 'winston';
```
Comment (pini85) [inline services/logger/prodLogger.js:1]: rename this as logger.js and remove the commentout Logger.js. please no capital

File: index.js [hunk 4]
```diff
@@ -1,17 +1,23 @@
-import express from "express";
-import "./services/DB/mongoose.js";
-import cors from "cors";
-import router from "./components/movies/movie.routes.js";
+import express from 'express';
+import './services/DB/mongoose.js';
+import cors from 'cors';
+import logger from './services/logger/index.js';
+
+logger.info('text info', { meta: 1 });
```
Comment (pini85) [inline index.js:6]: why are we putting the logger types here in our index js?

File: index.js [hunk 5]
```diff
@@ -1,17 +1,23 @@
...
+logger.warn('text warn');
```
Comment (pini85) [inline index.js:7]: also i dont see you implement Morgan with winston here.

**Review by pini85 — CHANGES_REQUESTED** [review]

File: index.js (hunk 4 above)
Comment (ReaGol) [reply index.js:6]: where do they need to be? did not see any source explaining that

File: services/logger/prodLogger.js (hunk 1 above)
Comment (ReaGol) [reply services/logger/prodLogger.js:20]: forgot to change it

File: index.js (hunk 4 above)
Comment (pini85) [reply index.js:6]: where did you see we need to explicitly write it out? We are doing it in the settings at logger.js when we create a logger instance

**Pull request merged**